src/bika/coa/reportview.py

```diff
@@ -845,7 +846,24 @@ def get_formatted_uncertainty(self, analysis):
         if uncertainty:
             return "&plusmn; {}".format(uncertainty)
         return
-    
+
+    def get_formatted_specs_hydro(self, model, analysis):
+        specs = analysis.getResultsRange()
+        fs = ''
+        precision = analysis.getPrecision()
+        precision_formatting = "{:." + str(precision) + "f}"
```

Comment:
have you tried using the python function round? - round(float, precision). Other it looks good, also maybe it is something that can be applied on the specification analysis view